### PR TITLE
Add instructions to readme to clarify steps

### PR DESCRIPTION
**Pull request by Lholbourn**
Instruct user on how to manually run jenkins job. Also adds why publishLocal can be useful for local development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
-This project generates, packages and deploys to the sonatype nexus the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project.
+This project generates, packages and deploys the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project to the sonatype nexus.
 
-These classes will be used by other TDR repositories to communicate with the consignment API.
+These classes are be used by other TDR repositories to communicate with the consignment API.
 
 ### Building locally
-* Add a new query file to `src/main/graphql`
+* If you add a new query file to `src/main/graphql`:
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
```
Comment (MancunianSam) [inline README.md]: Why have you put `If you add` here?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,24 @@
 ## TDR Generated GraphQL
 
-This project generates, packages and deploys to the sonatype nexus the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project.
+This project generates, packages and deploys the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project to the sonatype nexus.
 
-These classes will be used by other TDR repositories to communicate with the consignment API.
+These classes are be used by other TDR repositories to communicate with the consignment API.
```
Comment (suzannehamilton) [inline README.md]: Typo: "are be"

File: README.md [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
 
 ### Building locally
-* Add a new query file to `src/main/graphql`
+* If you add a new query file to `src/main/graphql`:
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
-* Run `sbt package publishLocal`
+* Run `sbt package publishLocal` - This allows you to use your updated version of the API locally to facilitate work on FE (This helps as sometimes Jenkins can be slow to build and catch up etc.)
```
Comment (suzannehamilton) [inline README.md]: The main reason to do this is because you want to check the changes work locally before you get them reviewed and published, rather than because of Jenkins. By the way, I think it's Sonatype that's the slow part. @MancunianSam might know more.

File: README.md [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
 
 ### Releasing
-* Commit and push the new query file in `src/main/graphql`
-* Merge to master. This project will be built by the deploy job for the consignment API but it can be also run manually. Go to the TDR Graphql Code Generation
-job in Jenkins, paste the schema from the consignemnt api repo into the build parameters and build.
+* Commit and push the new query file that sits in `src/main/graphql`
+* Merge to master.
+* This project will be built by the deploy job for the consignment API but can be also run manually.
```
Comment (suzannehamilton) [inline README.md]: This isn't currently true - we've got a card in the backlog to fix it, but for now could you update this to say that you have to run the deploy job manually?

File: README.md [hunk 5]
```diff
@@ -1,18 +1,24 @@
...
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
-* Run `sbt package publishLocal`
+* Run `sbt package publishLocal` - This allows you to use your updated version of the API locally to facilitate work on FE (This helps as sometimes Jenkins can be slow to build and catch up etc.)
 
 ### Releasing
-* Commit and push the new query file in `src/main/graphql`
-* Merge to master. This project will be built by the deploy job for the consignment API but it can be also run manually. Go to the TDR Graphql Code Generation
-job in Jenkins, paste the schema from the consignemnt api repo into the build parameters and build.
+* Commit and push the new query file that sits in `src/main/graphql`
+* Merge to master.
```
Comment (suzannehamilton) [inline README.md]: I know this is existing wording, but could you reword this to something like "Open a pull request, and merge to master once reviewed" so that it's clear that we still do PRs for this project rather than merging directly?

File: README.md [hunk 6]
```diff
@@ -1,18 +1,24 @@
...
+* This project will be built by the deploy job for the consignment API but can be also run manually.
+
+#####To Run Jenkins Manually:
```
Comment (suzannehamilton) [inline README.md]: Space after the hashes, and titles don't need a colon. Also, this looks like it should be four hashes rather than five.

File: README.md (hunk 3 above)
Comment (MancunianSam) [reply README.md]: You're right, it is because you want to check your work locally first and yes, it's the Sonatype deploy that takes the time.

File: README.md (hunk 2 above)
Comment (Lholbourn) [reply README.md]: Whoops! Will fix this!

File: README.md (hunk 3 above)
Comment (Lholbourn) [reply README.md]: Cool, I'll change that to reflect this!

File: README.md (hunk 1 above)
Comment (Lholbourn) [reply README.md]: I don't fully remember exactly why I did, I think it sounds better without it anyway so will change it back 👌
